### ISSUE TITLE
java.lang.ArrayIndexOutOfBoundsException in E-MEXP-748

### ISSUE THREAD

**Issue by alf239** [gxa/gxa#48]
```
2011-07-05 15:49:28,673 ERROR [pool-7-thread-5] loader.DefaultAtlasLoader$1 (DefaultAtlasLoader.java:122) - Loading error
java.lang.ArrayIndexOutOfBoundsException
        at ucar.ma2.Index3D.setDim(Index3D.java:100)
        at ucar.ma2.Index.set(Index.java:664)
        at uk.ac.ebi.gxa.netcdf.generator.NetCDFCreator.writeMap(NetCDFCreator.java:739)
        at uk.ac.ebi.gxa.netcdf.generator.NetCDFCreator.writeSampleOntologies(NetCDFCreator.java:705)
        at uk.ac.ebi.gxa.netcdf.generator.NetCDFCreator.write(NetCDFCreator.java:463)
        at uk.ac.ebi.gxa.netcdf.generator.NetCDFCreator.createNetCdf(NetCDFCreator.java:858)
        at uk.ac.ebi.gxa.loader.service.AtlasNetCDFUpdaterService.writeNetCDF(AtlasNetCDFUpdaterService.java:155)
        at uk.ac.ebi.gxa.loader.service.AtlasNetCDFUpdaterService.process(AtlasNetCDFUpdaterService.java:69)
        at uk.ac.ebi.gxa.loader.DefaultAtlasLoader$1$1.process(DefaultAtlasLoader.java:104)
        at uk.ac.ebi.gxa.loader.UpdateNetCDFForExperimentCommand.visit(UpdateNetCDFForExperimentCommand.java:39)
        at uk.ac.ebi.gxa.loader.DefaultAtlasLoader$1.run(DefaultAtlasLoader.java:75)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:441)
        at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
        at java.util.concurrent.FutureTask.run(FutureTask.java:138)
        at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
        at java.lang.Thread.run(Thread.java:662)
2011-07-05 15:49:28,674 ERROR [pool-7-thread-5] tasks.LoaderTask$1 (LoaderTask.java:159) - Task failed because of:
java.lang.ArrayIndexOutOfBoundsException
        at ucar.ma2.Index3D.setDim(Index3D.java:100)
        at ucar.ma2.Index.set(Index.java:664)
        at uk.ac.ebi.gxa.netcdf.generator.NetCDFCreator.writeMap(NetCDFCreator.java:739)
        at uk.ac.ebi.gxa.netcdf.generator.NetCDFCreator.writeSampleOntologies(NetCDFCreator.java:705)
        at uk.ac.ebi.gxa.netcdf.generator.NetCDFCreator.write(NetCDFCreator.java:463)
        at uk.ac.ebi.gxa.netcdf.generator.NetCDFCreator.createNetCdf(NetCDFCreator.java:858)
        at uk.ac.ebi.gxa.loader.service.AtlasNetCDFUpdaterService.writeNetCDF(AtlasNetCDFUpdaterService.java:155)
        at uk.ac.ebi.gxa.loader.service.AtlasNetCDFUpdaterService.process(AtlasNetCDFUpdaterService.java:69)
        at uk.ac.ebi.gxa.loader.DefaultAtlasLoader$1$1.process(DefaultAtlasLoader.java:104)
        at uk.ac.ebi.gxa.loader.UpdateNetCDFForExperimentCommand.visit(UpdateNetCDFForExperimentCommand.java:39)
        at uk.ac.ebi.gxa.loader.DefaultAtlasLoader$1.run(DefaultAtlasLoader.java:75)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:441)
        at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
        at java.util.concurrent.FutureTask.run(FutureTask.java:138)
        at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
        at java.lang.Thread.run(Thread.java:662)
```

**Comment by alf239** [comment]
Same problem with `E-GEOD-2666`, `E-MEXP-903`, `E-TABM-169`; it seems it's a common problem with all ontology-mapped sample properties.

**Comment by alf239** [comment]
The value is weird here, `[EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317, EFO_0000317]`

**Comment by alf239** [comment]
Fixed in 74168b4b5f4cec67c0fcca549f5bc9f8247e590f
